pkg/exporters/k8sexporter/condition/manager_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	problemutil "k8s.io/node-problem-detector/pkg/util"
 
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/util/clock"
+	clock "k8s.io/utils/clock/testing"
```

Comment:
Should be consistent across project.
```suggestion
	testclock "k8s.io/utils/clock/testing"
```